src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/GenericChipDeviceListener.kt

```diff
@@ -39,7 +39,7 @@ open class GenericChipDeviceListener : ChipDeviceController.CompletionListener {
     // No op
   }
 
-  override fun onOpCSRGenerationComplete(errorCode: ByteArray) {
+  override fun onOpCSRGenerationComplete(csr: ByteArray) {
```

Comment:
Is it the whole CSRResponse TLV payload (incl signature) or just the CSR and not the signature?